python/python/glide/async_commands/cluster_commands.py

```diff
@@ -150,7 +151,7 @@ async def client_id(
         Returns:
             TClusterResponse[int]: The id of the client.
             If a single node route is requested, returns a int representing the client's id.
-            Otherwise, returns a dict of [str , int] where each key contains the address of
+            Otherwise, returns a dict of [bytes , int] where each key contains the address of
```

Comment:
I don't think that IP address or DNS hostname could be a binary string. Please, revert.
It was also planned to fix GLIDE core to use `SimpleString` for that instad of `BulkString`

Reply:
So once this is fixed in Glide core we can revert it here.